src/scripts/release.zig

```diff
@@ -435,8 +449,39 @@ fn publish(shell: *Shell, languages: LanguageSet, info: VersionInfo) !void {
     }
 }
 
+fn last_changelog_entry(changelog: []const u8) []const u8 {
```

Comment:
Maybe s/last with "latest" or "newest" or "top"? "Last" could also mean the last in the file, which is the oldest.

Reply:
Yeah, totally should be latest, thanks!